### PR TITLE
Update kicad with proper artifact install location and add uninstall and zap stanzas

### PR DESCRIPTION
**Pull request by decrement**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/kicad.rb [hunk 1]
```diff
@@ -9,10 +9,14 @@
   homepage 'http://kicad-pcb.org/'
 
   suite 'Kicad-apps', target: 'Kicad'
-  artifact 'kicad', target: "#{ENV['HOME']}/Library/Application Support/kicad"
+  artifact 'kicad', target: '/Library/Application Support/kicad'
```
Comment (vitorgalvao) [inline Casks/kicad.rb]: Why?

Comment (decrement) [inline Casks/kicad.rb]: The correct location of the `kicad` directory placement is `/Library/Application Support/`, as evident by the symlink/alias provided in the DMG. Furthermore, the `KISYSMOD` environmental variable defaults to `/Library/Application Support/` as well, which is used by kicad-extras to locate the offline footprints. Of course, the end-user is able to redefine `KISYSMOD`, but what I am proposing is what I think the KiCad team intended and most friendly for the end-user.

**Comment by decrement** [comment]
The `brew cask uninstall kicad` did not remove the directory `/Library/Application Support/kicad/` without this line (that is why I included it):
```
uninstall delete: '/Library/Application Support/kicad'
```

**Comment by vitorgalvao** [comment]
> The `brew cask uninstall kicad` did not remove the directory `/Library/Application Support/kicad/`

Just tested it and can confirm, but that is clearly a bug. `uninstall` even says:

```
==> Uninstalling Cask kicad
==> Removing App Suite '/Applications/Kicad'.
==> Removing Generic Artifact '/Library/Application Support/kicad'.
==> Using sudo to gain ownership of path '/Library/Application Support/kicad'
```

And then it doesn’t remove it. The solution is not to add an `uninstall`, but to fix the bug. Pinging @reitermarkus.

**Comment by decrement** [comment]
I have updated the formula to use Ruby methods.

**Comment by decrement** [comment]
I found the reason for the failure in removing the Generic Artifact; I have submitted a pull request: Homebrew/brew#2587

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**